batch.md

```diff
@@ -108,8 +146,8 @@ env.fromCollection(inputData)
 env.execute("...");
 ```
 
-### Parameters
-A builder API is provided to construct an instance of `FlinkPravegaOutputFormat`. See the table below for a summary of builder properties.  Note that the builder accepts an instance of `PravegaConfig` for common configuration properties.  See the [configurations](configurations) page for more information.
+### Builder API Usage

```

Comment:
Can we keep the heading as `Parameters`?

Reply:
Accepted suggestion